components/email/src/Email.svelte

```diff
@@ -863,6 +864,58 @@
       ? MessageType.DRAFTS
       : MessageType.MESSAGES;
   }
+
+  interface DisplayedParticipant extends Participant {
+    _type: "to" | "cc" | "bcc";
+  }
+  /**
+   * Returns an aggregated list of participants including their _type.
+   */
+  function getAllRecipients({
+    to,
+    cc,
+    bcc,
+  }: Record<"to" | "cc" | "bcc", Participant[]>): Promise<
+    DisplayedParticipant[]
+  > {
+    return Promise.resolve([
```

Comment:
Why is this a Promise?

Reply:
sveltes `#await` expects a Promise so I'm being explicit here rather than implicitly returning.